txp/Pos/Script.hs

```diff
@@ -154,6 +159,8 @@ stdlib = case PL.loadLibrary PL.emptyDeclContext prelude of
   where
     prelude = $(lift . toString =<< runIO PL.preludeString)
 
+deriving instance TH.Lift Script
```

Comment:
Why is it here and not in the place where `Script` is defined?

Reply:
this is revenge for all my orphans

Reply:
I thought it was some type from `plutus-prototype` or whatever, I'll push a fix in a moment.

Reply:
If it was from `plutus-prototype`, I'd write something like this 😉 
https://i.imgur.com/ydqOOeD.png